target_chains/solana/programs/pyth-solana-receiver/src/state/config.rs

```diff
@@ -0,0 +1,23 @@
+use {
+    anchor_lang::prelude::*,
+    solana_program::pubkey::Pubkey,
+};
+
+#[account]
+pub struct Config {
+    pub governance_authority:          Pubkey,
+    pub target_governance_authority:   Option<Pubkey>,
+    pub wormhole:                      Pubkey,
+    pub valid_data_sources:            Vec<DataSource>,
+    pub single_update_fee_in_lamports: u64,
+}
+
+#[derive(AnchorSerialize, AnchorDeserialize, Clone)]
+pub struct DataSource {
+    pub chain:   u16,
+    pub emitter: Pubkey,
+}
+
+impl Config {
+    pub const LEN: usize = 1000; // PLACEHOLDER LEN, DECIDE HOW BIG WE WANT TO MAKE IT
+}
```

Comment:
this seems like the kind of thing we can forget to update later. 1000 is definitely big enough right now, but could get tight if we add to the struct.

maybe add a test where you make a Config struct with some reasonable values (like 2 data sources etc) then assert that the LEN > the size of that struct? Or maybe even the size of the struct * 2 or something? 

(I'm not convinced about the idea ^. Main thing is let's not forget to update this later.)